arch/rv32i/src/pmp.rs

```diff
@@ -0,0 +1,227 @@
+use kernel;
```

Comment:
Need `//!` comment.

Reply:
Where/why?

Reply:
It's a Rust-ism and is useful for the generated docs. Most files should have them, for instance: https://github.com/tock/tock/blob/master/arch/cortex-m3/src/mpu.rs#L1

Reply:
Oops--fixed.